Userland/Applications/Calendar/main.cpp

```diff
@@ -105,6 +107,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (default_view == "Year")
         view_year_action->set_checked(true);
 
+    auto open_settings_action = GUI::Action::create("&Settings", {}, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/app-settings.png"sv)), [&](const GUI::Action&) {
```

Comment:
`const GUI::Action&` => `GUI::Action const&`. One day clang-format will learn to do this properly. :^)